RecoTracker/IterativeTracking/python/LowPtTripletStep_cff.py

```diff
@@ -274,12 +277,17 @@
 lowPtTripletStep.mva.GBRForestLabel = 'MVASelectorIter1_13TeV'
 lowPtTripletStep.qualityCuts = [-0.6,-0.3,-0.1]
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(lowPtTripletStep, TrackMVAClassifierPrompt.clone(
```

Comment:
note that the pre-DNN version was cloning from lowPtTripletStep and pp_on_AA and highBetaStar were simpler toModify

Reply:
Changed back to clone from lowPtTripletStep